apps/sms/index.html

```diff
@@ -128,6 +129,7 @@
   </head>
 
   <body role="application" class="skin-comms theme-communications">
+    <iframe src="/views/shared/shim_host.html" class="shim-host" height="0" width="0"></iframe>
```

Comment:
can you please measure with raptor if adding the iframe changes the launch time ? Actually I'm sure it does and I'm also sure it's better to have it in the markup so that we have the shims ready at startup time, but it's good to know how much we'll regress because of this.